File: src/main/java/com/atsuishio/superbwarfare/item/NetheriteHammer.java
package com.atsuishio.superbwarfare.item;

import net.minecraft.world.item.Tiers;

public class NetheriteHammer extends Hammer {

    public NetheriteHammer() {
        super(Tiers.NETHERITE, 13, -3.2f, new CustomDamageProperty(false).fireResistant());
    }

}

File: src/main/java/com/atsuishio/superbwarfare/item/NetheriteHammer.kt
package com.atsuishio.superbwarfare.item

import net.minecraft.world.item.ItemStack
import net.minecraft.world.item.Tiers

class NetheriteHammer : Hammer(Tiers.NETHERITE, 13, -3.2f, Properties().durability(2800).fireResistant()) {
    override fun isDamageable(stack: ItemStack?): Boolean {
        return false
    }
}
